src/utils.ts

```diff
@@ -1419,13 +1419,28 @@ export function enumToString(en: Record<string, unknown>): string {
  *
  * @internal
  */
-export function supportsRetryableWrites(server: Server): boolean {
-  return (
-    !!server.loadBalanced ||
-    (server.description.maxWireVersion >= 6 &&
-      !!server.description.logicalSessionTimeoutMinutes &&
-      server.description.type !== ServerType.Standalone)
-  );
+export function supportsRetryableWrites(server?: Server): boolean {
+  if (!server) {
+    return false;
+  }
+
+  if (server.loadBalanced) {
+    // Loadbalanced topologies will always support retry writes
+    return true;
+  }
+
+  if (server.description.maxWireVersion >= 6) {
```

Comment:
do we really need this check considering our min supported wire version is 6?

Reply:
Nope good point! removed!